armi/reactor/components/__init__.py

```diff
@@ -341,6 +341,23 @@ def computeVolume(self):
         """
         return self._deriveVolumeAndArea()
 
+    @staticmethod
+    def getMaxVolume(block):
+        """
+        The maximum volume of the given Block if it were totally full.
+
+        Parameters
+        ----------
+        block : Block
+            Block of interest.
+
+        Returns
+        -------
+        vol : float
+            volume in cm^3.
+        """
+        return block.getMaxArea() * block.getHeight()
```

Comment:
Thoughts? Clearly you could need to make this not a `staticmethod`, but I think it makes sense since this method is only called in the context when a DerivedShape component has a parent block. 
```suggestion
        return self.parent.getMaxArea() * self.parent.getHeight()
```

Reply:
There is one place downsteram where this is used as a static method, so it'll be less work to just keep it that way.

Reply:
done